src/function/table/sqlite/pragma_version.cpp

```diff
@@ -0,0 +1,39 @@
+#include "duckdb/function/table/sqlite_functions.hpp"
+
+namespace duckdb {
+
+struct PragmaVersionData : public TableFunctionData {
+	PragmaVersionData() : done(false) {
+	}
+	bool done;
+};
+
+static unique_ptr<FunctionData> pragma_version_bind(ClientContext &context, vector<Value> inputs,
+                                                    vector<SQLType> &return_types, vector<string> &names) {
+	names.push_back("library_version");
+	return_types.push_back(SQLType::VARCHAR);
+    names.push_back("source_id");
+    return_types.push_back(SQLType::VARCHAR);
+
+	return make_unique<PragmaVersionData>();
+}
+
+static void pragma_version_info(ClientContext &context, vector<Value> &input, DataChunk &output,
+                                FunctionData *dataptr) {
+	auto &data = *((PragmaVersionData *)dataptr);
+	assert(input.size() == 0);
+	if (data.done) {
+		// finished returning values
+		return;
+	}
+	output.SetCardinality(1);
+	output.SetValue(0, 0, "DuckDB");
+	output.SetValue(1, 0, DUCKDB_SOURCE_ID);
```

Comment:
this is defined in the `CMakeLists.txt` file, howe it's also lives in the `sqlite_api_wrapper.cpp` file: https://github.com/cwida/duckdb/blob/7621c51e48bec80b61bede7f5fc5729ec4d09b76/tools/sqlite3_api_wrapper/sqlite3_api_wrapper.cpp#L694-L699

Originally I had tried to `include <sqlite3.h>` and use the `sqlite3_libversion` and `sqlite3_source_id` functions. The linker couldn't find the `sqlite3_libversion` and `sqlite3_source_id` symbols, I tried to change various CMakeLists.txt to make it work, but I couldn't find quite the right incantation, so I ended up duplicating the functionality here.  Ideally we would be able to use those functions.